scripts/user/setup-test-users.ts

```diff
@@ -93,74 +93,68 @@ const setupPetitioners = async (
-      'RT9999',
-      'RT6666',
-      'RT0000',
-      'RT1111',
-      'RT2222',
-      'RT3333',
-    ],
-    privatePractitioner: [
-      'PT1234',
-      'PT5432',
-      'PT1111',
-      'PT2222',
-      'PT3333',
-      'PT4444',
-      'PT5555',
-      'PT6666',
-      'PT7777',
-      'PT8888',
-    ],
-  };
-
-  for (let role in practitioners) {
-    const promises = practitioners[role].map((barNumber, i) => {
-      const practiceType = role === 'privatePractitioner' ? 'Private' : 'IRS';
-      const email = `${role}${i + 1}@example.com`;
-      const user = {
-        admissionsDate: '2019-03-01',
-        admissionsStatus: 'Active',
-        barNumber,
-        birthYear: '1950',
-        contact: {
-          address1: '234 Main St',
-          address2: 'Apartment 4',
-          address3: 'Under the stairs',
-          city: 'Chicago',
-          countryType: 'domestic',
-          phone: '+1 (555) 555-5555',
-          postalCode: '61234',
-          state: 'IL',
-        },
-        email,
-        firmName: 'Some Firm',
-        firstName: `${role} ${i + 1}`,
-        lastName: 'Test',
-        name: `Test ${role}${i + 1}`,
-        originalBarState: 'WA',
-        password: DEFAULT_ACCOUNT_PASS,
-        practiceType,
-        practitionerType: 'Attorney',
-        role,
-        section: role,
-        suffix: '',
-      };
-
-      return createOrUpdateUser(applicationContext, {
-        password: DEFAULT_ACCOUNT_PASS!,
-        setPasswordAsPermanent: true,
-        user,
-      });
-    });
+  const allPractitioners = [
+    { barNumber: 'RT6789', role: 'irsPractitioner' },
+    { barNumber: 'RT0987', role: 'irsPractitioner' },
+    { barNumber: 'RT7777', role: 'irsPractitioner' },
+    { barNumber: 'RT8888', role: 'irsPractitioner' },
+    { barNumber: 'RT9999', role: 'irsPractitioner' },
+    { barNumber: 'RT6666', role: 'irsPractitioner' },
+    { barNumber: 'RT0000', role: 'irsPractitioner' },
+    { barNumber: 'RT1111', role: 'irsPractitioner' },
+    { barNumber: 'RT2222', role: 'irsPractitioner' },
+    { barNumber: 'RT3333', role: 'irsPractitioner' },
+    { barNumber: 'PT1234', role: 'privatePractitioner' },
+    { barNumber: 'PT5432', role: 'privatePractitioner' },
+    { barNumber: 'PT1111', role: 'privatePractitioner' },
+    { barNumber: 'PT2222', role: 'privatePractitioner' },
+    { barNumber: 'PT3333', role: 'privatePractitioner' },
+    { barNumber: 'PT4444', role: 'privatePractitioner' },
+    { barNumber: 'PT5555', role: 'privatePractitioner' },
+    { barNumber: 'PT6666', role: 'privatePractitioner' },
+    { barNumber: 'PT7777', role: 'privatePractitioner' },
+    { barNumber: 'PT8888', role: 'privatePractitioner' },
+  ];
+
+  for (let i = 0; i < allPractitioners.length; i++) {
+    const { barNumber, role } = allPractitioners[i];
 
-    await Promise.all(promises);
+    const practiceType = role === 'privatePractitioner' ? 'Private' : 'IRS';
+    const email = `${role}${i + 1}@example.com`;
```

Comment:
You are now doing all of the practitioners in the same loop.

This means you're creating irsPractitioner1 through irsPractitioner10 and privatePractitioner11 through privatePractitioner20.

Are we gonna clean up the old privatePractitioner1 through privatePractitioner10 records?

Are we gonna remove all instances of privatePractitioner < 11 in various tests?

Reply:
Great catch! Fixed it!